sdk/core/core-http/lib/policies/proxyPolicy.ts

```diff
@@ -12,23 +12,90 @@ import { ProxySettings } from "../serviceClient";
 import { WebResource } from "../webResource";
 import { Constants } from "../util/constants";
 import { URLBuilder } from "../url";
+let bypass: any = null;
+
+function environmentProxyValue(name: string): string | undefined {
```

Comment:
This is a generic "get env variable" method that shouldn't be limited to Proxy values.  Please find a place for it in utils.

Reply:
Make sure something like this doesn't already exist.